packages/bigcommerce-payments-integration/src/components/BigCommercePaymentsPaymentMethodComponent.test.tsx

```diff
@@ -0,0 +1,429 @@
+import {
+    AccountInstrument,
+    createCheckoutService,
+    HostedInstrument,
+    LanguageService,
+} from '@bigcommerce/checkout-sdk';
+import { render } from '@testing-library/react';
+import { EventEmitter } from 'events';
+import React from 'react';
+
+import { PaymentFormService } from '@bigcommerce/checkout/payment-integration-api';
+import { getPaymentFormServiceMock } from '@bigcommerce/checkout/test-mocks';
+
+import { getBigCommercePaymentsMethod } from '../mocks/paymentMethods.mock';
+
+import BigCommercePaymentsPaypalPaymentMethodComponent from './BigCommercePaymentsPaypalPaymentMethodComponent';
+
+describe('BigCommercePaymentsPaypalPaymentMethodComponent', () => {
```

Comment:
As I understand this is generic component that used by others, so in this case it should be called as BigCommercePayments (without PayPal)
```suggestion
describe('BigCommercePaymentsPaymentMethodComponent', () => {
```